src/lib.rs

```diff
@@ -161,17 +161,17 @@ async fn launch_jsonrpc_server(
     let history_handler = history_handler.ok_or_else(|| {
         "History network must be available to use IPC transport for JSON-RPC server".to_string()
     })?;
-    match trin_config.web3_transport.as_str() {
-        "ipc" => {
-            // Launch jsonrpsee server with IPC transport
```

Comment:
This comment looks correct to me, `Web3TransportType::IPC` means we run the server with IPC transport, not HTTP and WS.

Reply:
Typo, my bad